command/sub8_missions/missions/pose_sequence_test.py

```diff
@@ -0,0 +1,24 @@
+from txros import util
+import numpy as np
+from mil_misc_tools import text_effects
+from sub8.sub_singleton import PoseSequenceCommander
+
+fprint = text_effects.FprintFactory(title="SEQUENCE_TEST").fprint
+
+@util.cancellableInlineCallbacks
+def run(sub_singleton):
+    '''Used for testing in sim. These commands may not be safe for the real sub.'''
+    fprint("Starting Test With Eulers", msg_color="blue")
+    positions = [np.array([-1, 0, 0]), np.array([0, 0, 3]),
```

Comment:
It'd be nice if I could just put normal ole' python arrays as the input to PoseSequenceCommander and it would convert to numpy for me so i don't have to type np.array a million times